cdk/lib/cache.ts

```diff
@@ -47,7 +47,7 @@ export class CacclCache extends Construct {
       vpcSecurityGroupIds: [sg.securityGroupId],
     });
 
-    sg.addIngressRule(Peer.anyIpv4(), Port.tcp(6379));
+    sg.addIngressRule(Peer.ipv4(vpc.vpcCidrBlock), Port.tcp(6379), 'elasticache instance');
```

Comment:
this was an omission during development. the cache runs on a private subnet anyway, but still.